pallets/gear-debug/src/tests.rs

```diff
@@ -284,6 +284,7 @@ fn get_last_message_id() -> MessageId {
     }
 }
 
+#[cfg(feature = "lazy-pages")]
```

Comment:
If I remember right, you said that no behaviour should be changed with the feature, and all tests must run out of this?

Reply:
Yep, remove this